### ISSUE TITLE
 Yangon - OSM Map Enhancement Effort 

### ISSUE THREAD

**Issue by GRABOSM** [GRABOSM/Grab-Data#20]
Hello Mappers,

We started working on Yangon city from 6th Aug'18, enhancing the road network in osm and focus only on areas served by Grab.

We have done pre-analysis on few areas in Yangon and compared with strava data and offset database to find if there are any offset imagery. 
- No offsets were identified.

Basis our analysis, digi standarad and Esri has latest image and are in sync with Gps Traces and existing data.

We will follow the general  OSM guidelines outlined in WIKI page, please let us know if there any country specific policies to be followed.

Thanks
Grab Team

**Comment by GRABOSM** [comment]
Hello All,

While reviewing Yangon, we identified instances where a few incorrect edits made by other mappers - incorrect connectives and duplicate segments.  We contacted both the mappers through their individual change-set comments.  While we are yet to receive a reply from them,this is our attempt to contact larger audience and seek guidance.

Here are the examples of incorrect edits we found in Yangon and/or edits we are not able to validate through existing resources we have in JOSM - 
Kindly take time to review and suggest
